src/components/navigation/Navigation.js

```diff
@@ -54,6 +54,10 @@ class Navigation extends Component {
         const desktopMenu = document.getElementById('desktop-menu');
         const mobileMenu = document.getElementById('mobile-menu');
 
+        if (e.target.tagName === 'SPAN') {
+            return false
```

Comment:
I don't think we need this since the 'refresh' icon is within the menu div, maybe I'm missing something? Also a side effect of using 'SPAN' here is that clicking on a span on the page that's outside of the menu won't close the menu now.

Reply:
So the `contains` doesn't seems to work with this element - I guess it's because it's added dynamically - the same is with switching accounts, it closes menu but it's not BUTTON either A. I will test it.